### PR TITLE
[D1] Adding a line of text to encourage horizontal scaling in D1

### PR DESCRIPTION
**Pull request by Oxyjun**
### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

**Comment by vy-ton** [comment]
@Oxyjun we should have something similar to https://developers.cloudflare.com/durable-objects/platform/limits/#how-much-work-can-a-single-durable-object-do for D1.

File: src/content/docs/d1/index.mdx [hunk 1]
```diff
@@ -25,6 +25,8 @@ Create new serverless SQL databases to query from your Workers and Pages project
 
 D1 is Cloudflare’s native serverless database. D1 allows you to build applications that handle large amounts of users at no extra cost. With D1, you can restore your database to any minute within the last 30 days.
 
+D1 is designed to be scaled out horizontally - we recommend creating multiple, smaller databases for your application.
```
Comment (vy-ton) [inline src/content/docs/d1/index.mdx]:
```suggestion
D1 is Cloudflare’s managed, serverless database with SQLite's SQL semantics, built-in disaster recovery, and Worker & HTTP API access. 

D1 is designed for horizontal scale out across multiple, smaller (10GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with 1000s of databases at no extra cost for isolating with multiple databases; D1 pricing is based only on query and storage costs.
```

File: src/content/docs/d1/index.mdx [hunk 2]
```diff
@@ -25,6 +25,10 @@ Create new serverless SQL databases to query from your Workers and Pages project
 
 D1 is Cloudflare’s native serverless database. D1 allows you to build applications that handle large amounts of users at no extra cost. With D1, you can restore your database to any minute within the last 30 days.
 
+D1 is Cloudflare’s managed, serverless database with SQLite's SQL semantics, built-in disaster recovery, and Worker & HTTP API access. 
+
+D1 is designed for horizontal scale out across multiple, smaller (10GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with 1000s of databases at no extra cost for isolating with multiple databases; D1 pricing is based only on query and storage costs.
```
Comment (Oxyjun) [inline src/content/docs/d1/index.mdx]:
```suggestion
D1 is designed for horizontal scale out across multiple, smaller (10 GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with thousands of databases at no extra cost for isolating with multiple databases; D1 pricing is based only on query and storage costs.
```

File: src/content/docs/d1/index.mdx [hunk 3]
```diff
@@ -23,7 +23,11 @@ Create new serverless SQL databases to query from your Workers and Pages project
 
 <Plan type="workers-all" />
 
-D1 is Cloudflare’s native serverless database. D1 allows you to build applications that handle large amounts of users at no extra cost. With D1, you can restore your database to any minute within the last 30 days.
+D1 is Cloudflare's native serverless database. D1 allows you to build applications that handle large amounts of users at no extra cost. With D1, you can restore your database to any minute within the last 30 days.
+
+D1 is Cloudflare's managed, serverless database with SQLite's SQL semantics, built-in disaster recovery, and Worker & HTTP API access.
```
Comment (vy-ton) [inline src/content/docs/d1/index.mdx]: this sentence should replace the paragraph. sorry my suggestion wasn't clear

File: src/content/docs/d1/platform/limits.mdx [hunk 4]
```diff
@@ -41,4 +41,24 @@ Limits for individual queries (listed above) apply to each individual statement
...
 
-<Render file="limits_increase" product="workers" />
\ No newline at end of file
+<Render file="limits_increase" product="workers" />
+
+## Frequently Asked Questions
+
+Frequently asked questions related to D1 limits:
+
+### How much data can I store in a D1 database?
```
Comment (vy-ton) [inline src/content/docs/d1/platform/limits.mdx]: I would combine both questions. And can we avoid reiterating the limits but instead provide user context on how they should approach their data design and archtiecture.

Expand on `D1 is designed for...` paragraph

Comment (vy-ton) [reply src/content/docs/d1/platform/limits.mdx]: This is where we should have similar content to https://developers.cloudflare.com/durable-objects/platform/limits/#how-much-work-can-a-single-durable-object-do

File: src/content/docs/d1/index.mdx (hunk 3 above)
Comment (Oxyjun) [reply src/content/docs/d1/index.mdx]: Ah I misunderstood - will edit.

File: src/content/docs/d1/platform/limits.mdx [hunk 5]
```diff
@@ -41,4 +41,20 @@ Limits for individual queries (listed above) apply to each individual statement
...
+
+Frequently asked questions related to D1 limits:
+
+### How much work can a D1 database do?
+
+D1 is designed for horizontal scale out across multiple, smaller (10 GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with thousands of databases at no extra cost for isolating with multiple databases, as the pricing is based only on query and storage costs.
+
+- Each D1 database can store up to 10 GB of data, and you can create up to 50,000 separate D1 databases. This allows you to split a single monolithic database into multiple, smaller databases.
+- Working with smaller databases may allow you to query your databases more efficiently, and ultimately result in higher query throughput.
+- Smaller databases may require simpler query statements, reducing the processing time to return your query.
```
Comment (vy-ton) [inline src/content/docs/d1/platform/limits.mdx]: I would remove this line

File: src/content/docs/d1/platform/limits.mdx [hunk 6]
```diff
@@ -41,4 +41,20 @@ Limits for individual queries (listed above) apply to each individual statement
...
+## Frequently Asked Questions
+
+Frequently asked questions related to D1 limits:
+
+### How much work can a D1 database do?
+
+D1 is designed for horizontal scale out across multiple, smaller (10 GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with thousands of databases at no extra cost for isolating with multiple databases, as the pricing is based only on query and storage costs.
+
+- Each D1 database can store up to 10 GB of data, and you can create up to 50,000 separate D1 databases. This allows you to split a single monolithic database into multiple, smaller databases.
+- Working with smaller databases may allow you to query your databases more efficiently, and ultimately result in higher query throughput.
```
Comment (vy-ton) [inline src/content/docs/d1/platform/limits.mdx]: things to note here would be stronger data isolation, queries over smaller working data execute faster

avoid the mention of throughput

File: src/content/docs/d1/platform/limits.mdx [hunk 7]
```diff
@@ -41,4 +41,20 @@ Limits for individual queries (listed above) apply to each individual statement
...
+
+## Frequently Asked Questions
+
+Frequently asked questions related to D1 limits:
+
+### How much work can a D1 database do?
+
+D1 is designed for horizontal scale out across multiple, smaller (10 GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with thousands of databases at no extra cost for isolating with multiple databases, as the pricing is based only on query and storage costs.
+
+- Each D1 database can store up to 10 GB of data, and you can create up to 50,000 separate D1 databases. This allows you to split a single monolithic database into multiple, smaller databases.
```
Comment (vy-ton) [inline src/content/docs/d1/platform/limits.mdx]:
```suggestion
- Each D1 database can store up to 10 GB of data, and you can create up to thousands of separate D1 databases. This allows you to split a single monolithic database into multiple, smaller databases, thereby isolating application data by user, customer, or tenant.
```

File: src/content/docs/d1/platform/limits.mdx (hunk 6 above)
Comment (vy-ton) [inline src/content/docs/d1/platform/limits.mdx]:
```suggestion
- SQL queries over a smaller working data set can be more efficient and performant.
```
Suggestion to combine this bullet and the one below

File: src/content/docs/d1/index.mdx [hunk 8]
```diff
@@ -23,7 +23,9 @@ Create new serverless SQL databases to query from your Workers and Pages project
 
 <Plan type="workers-all" />
 
-D1 is Cloudflare’s native serverless database. D1 allows you to build applications that handle large amounts of users at no extra cost. With D1, you can restore your database to any minute within the last 30 days.
+D1 is Cloudflare's managed, serverless database with SQLite's SQL semantics, built-in disaster recovery, and Worker & HTTP API access.
```
Comment (marciocloudflare) [inline src/content/docs/d1/index.mdx]:
```suggestion
D1 is Cloudflare's managed, serverless database with SQLite's SQL semantics, built-in disaster recovery, and Worker and HTTP API access.
```

File: src/content/docs/d1/index.mdx [hunk 9]
```diff
@@ -23,7 +23,9 @@ Create new serverless SQL databases to query from your Workers and Pages project
...
+
+D1 is designed for horizontal scale out across multiple, smaller (10 GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with thousands of databases at no extra cost for isolating with multiple databases; D1 pricing is based only on query and storage costs.
```
Comment (marciocloudflare) [inline src/content/docs/d1/index.mdx]:
```suggestion
D1 is designed for horizontal scale out across multiple, smaller (10 GB) databases, such as per-user, per-tenant or per-entity databases. D1 allows you to build applications with thousands of databases at no extra cost for isolating with multiple databases. D1 pricing is based only on query and storage costs.
```

**Review by marciocloudflare — APPROVED** [review]

**Review by vy-ton — APPROVED** [review]

**Pull request merged**